test/e2e/update-feeds.feature.ts

```diff
@@ -10,5 +10,5 @@ it('reads blockchain data correctly', async () => {
 
   await runUpdateFeed(providerName, chain, chainId);
 
-  expect(logger.debug).toHaveBeenNthCalledWith(2, 'Processing batch of active dAPIs', expect.anything());
+  expect(logger.debug).toHaveBeenCalledWith('Fetching first batch of dAPIs batches');
```

Comment:
This is not testing what it should. Because we don't have the full logic implemented, I added a log statement that calls the `processBatch` which means that the blockchain state was read (and decoded) correctly.